config/NOTES.txt

```diff
@@ -0,0 +1,4 @@
+External charts config files are prefixed with "ext_" to be easily noticed.
+
+In these config, we're setting/surcharging and updating the image.tag with udpatecli (See ./updatecli/updatecli.d/docker-images).
```

Comment:
s/udpatecli/updatecli

Reply:
Grrr... I have to find a tool/trick to avoid this error which bitten me more than once already...

Reply:
https://pre-commit.com/index.html with codespell plugin maybe?

Reply:
I've got the codespell plugin already, reading the pre-commit link, thanks for the suggestion @halkeye :)